lib/fizzy/execute.cpp

```diff
@@ -1014,7 +1014,7 @@ ExecutionResult execute(Instance& instance, FuncIdx func_idx, span<const Value>
         case Instr::i32_eqz:
         {
             const auto value = static_cast<uint32_t>(stack.pop());
-            stack.push(value == 0);
+            stack.push(uint32_t{value == 0});
```

Comment:
Why do we need to make this clear? I know eqz returns 32-bit but we said we don't care because it is a union.

Reply:
Does this have any effect on speed?

Reply:
The `Value(bool)` is deleted to avoid some surprises. So the bool expression must be converted to `uint32_t` (or `uint64_t`, but `uint32_t` seems nicer).

Reply:
> Does this have any effect on speed?

Nope.